### ISSUE TITLE
Some pages of my library are not translated when using setTranslation()

### ISSUE THREAD

**Issue by SparkMonkey** [ngneat/transloco#150]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[x] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

I applied the strategy described [in this issue](https://github.com/ngneat/transloco/issues/144) to translate my libraries.

Example of a module inside my library : 

    import * as en from './i18n/en.json';

    ...

    constructor(private translocoService: TranslocoService) {
        translocoService.setTranslation(en, 'en');
    }

Sadly, some modules are not translated (11/28 on the concerned library).

The result is some pages are not translated at all. Some pages are partially translated because they pull different components from different modules of the same library.

Some toastr are translated whereas some other aren't.

I checked that there is no i18n key conflict across my i18n files.

The integration of transloco is, according to me, the same for every modules (described above)

I don't understand what can produce this behaviour. I'm hoping this is not a bug or some kind of conflict with an other library 🙏 

From this behaviour, do you have an idea where can it come from ?

Thank you

## Expected behavior

All components translated 👍 

## Environment

<pre><code>
Angular version: 8.2.13

Browser:
- [x] Chrome (desktop) version 77
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [x] Firefox version 70
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: 12.13.0
- Platform:  Windows

Others:
Using Angular Cli to package libraries.
</code></pre>

**Comment by NetanelBasal** [comment]
Can you reproduce it somehow? If not, I would put a debugger and step into the `setTranslation` method.

**Comment by SparkMonkey** [comment]
I can't reproduce this since I do not see any difference between modules.

Following your advise, I debugged by adding the following config in angular.json to be able to set breakpoint in the browser.

        "serve": {
          "builder": "@angular-devkit/build-angular:dev-server",
          "options": {
            ...
            "sourceMap": {
              "scripts": true,
              "styles": true,
              "vendor": true
            }
          },

I found the probable cause of the issue in the arguments of the following function : 

`setTranslation(translation: Translation, lang = this.getActiveLang(), options: SetTranslationOptions = {})` 

The 'translation' argument that represents the en.json have its keys prefixed by 'default' for defective modules.

**Example :**

en.json :

    {
        "errors": {
            "status": {
                "result": {
                    "title": "test",
                }
            }
        }
    }

**Working example :** 

Key in the 'translation' argument : errors.status.result.title

**Defective example :** 

Key in the 'translation' argument : default.errors.status.result.title

Can the result of `import * as en from './i18n/en.json';` be inconstant ?

Thank you
